PanRangeSlider/Common/RangeSlider.cs

```diff
@@ -31,6 +31,71 @@ public class RangeSlider : TemplatedView
 
     public event EventHandler? UpperDragCompleted;
 
+    public static BindableProperty ThumbCornerRadiusProperty =
+    BindableProperty.Create(nameof(ThumbCornerRadius), typeof(double), typeof(RangeSlider), 10.0, propertyChanged: OnLayoutPropertyChanged);
+
+public static BindableProperty LowerThumbCornerRadiusProperty =
+    BindableProperty.Create(nameof(LowerThumbCornerRadius), typeof(double), typeof(RangeSlider), -1.0, propertyChanged: OnLayoutPropertyChanged);
+
+public static BindableProperty UpperThumbCornerRadiusProperty =
+    BindableProperty.Create(nameof(UpperThumbCornerRadius), typeof(double), typeof(RangeSlider), -1.0, propertyChanged: OnLayoutPropertyChanged);
+
+public double ThumbCornerRadius
+{
+    get => (double)GetValue(ThumbCornerRadiusProperty);
+    set => SetValue(ThumbCornerRadiusProperty, value);
+}
+
+public double LowerThumbCornerRadius
+{
+    get => (double)GetValue(LowerThumbCornerRadiusProperty);
+    set => SetValue(LowerThumbCornerRadiusProperty, value);
+}
+
+public double UpperThumbCornerRadius
+{
+    get => (double)GetValue(UpperThumbCornerRadiusProperty);
+    set => SetValue(UpperThumbCornerRadiusProperty, value);
+}
```

Comment:
Please make sure all instance properties are near other instance properties. I suggest to append them to the bottom